### DISCUSSION TITLE
Script to download Backups

### DISCUSSION THREAD

**Discussion by dikkadev** [typst/typst#1236] [Show and tell]
I like using the web version more than doing it locally, but good backups (or less good versioning) are essential.

So I made a script that downloads the backup from the project page, renames it (to the date and time), and sends you a notification.

https://gist.github.com/Sett17/a47a0c3eb131664809aac50837553fba

_Note: I have only tested the whole thing with Windows_

I run this 2 times a day via the Task Scheduler, the same could also quickly be done with cron or something similar.

**Comment by davidleejy** [comment]
This is actually a kinda neat idea!
